iati/core/tests/test_rulesets.py

```diff
@@ -387,12 +378,16 @@ def valid_dataset(self):
         return iati.core.tests.utilities.DATASET_FOR_DATEORDER_RULE_VALID
 
     @pytest.fixture(params=[
-        {"less": "planned-disbursement/period-start/@iso-date", "more": "planned-disbursement/period-end/@iso-date"}
+        {'less': 'planned-disbursement/period-start/@iso-date', 'more': 'planned-disbursement/period-end/@iso-date'}
```

Comment:
Needs a range of separate tests writing. Based on the params to `valid_case`, would expect something in the region of 12-20.